### ISSUE TITLE
Fix barometric setting increments on the ISIS

### ISSUE THREAD

**Issue by BlueberryKing** [flybywiresim/a32nx#5707]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2021-08-23T20:45:53+00:00",
  "ref": "HEAD",
  "sha": "10169b43abb8568044bab599783275b34f2bca70",
  "actor": "BlueberryKing <22713769+BlueberryKing@users.noreply.github.com>",
  "event_name": "manual"
}

^ This is a custom build but it doesn't matter
```

### Describe the bug

Right now, when you change the pressure on the ISIS, we increment the hPa value by 1 which changes the inHg value by 0.03. This does not allow for fine tuning of the inHg value.

Instead, we should change whatever the default pressure unit is set to in the EFB by one unit.

### Expected behavior

When the default pressure setting in the EFB is set to inHg, the baro knob should change the inHg value on the ISIS by 0.01.

### Steps to reproduce

1. Load up anywhere
2. Change the baro setting
3. Observe how the inHg value changes by 0.03

### Discord Username (optional)

BlueberryKing#6641

**Comment by tracernz** [comment]
Alternately, add the option (guess it's a pin program?) that shows imperial pressure on the ISIS to the EFB, and increment by 1 hPa when it's hPa only.

**Comment by BlueberryKing** [comment]
Yeah, I mean I personally don't really mind, but I don't really wanna clutter the EFB with options, but that's more of a personal opinion. We can have a chat about it.

**Comment by bouveng** [comment]
Just make it behave like the real one. Ticks of 1 unit not matter inHg or hPa?

**Comment by BlueberryKing** [comment]
Hmm, this is not clear to me. We obviously want both indications to match up with each other, but 0.01 inHg does not equal 1 hPa since there's a conversion factor that is not a power of 10. So like

29.92 inHg / 1013 hPa is fine, but
29.93 inHg / 1014 hPa doesn't make sense

Or did you mean something else?

**Comment by tracernz** [comment]
The knob should adjust the barametric pressure by 10 hPa per full rotation... not sure if it has steps around that rotation and if so how many there are.
